### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by mesielepush**
Pull request to test my testing abilities on Jest

### CODE REVIEW

File: test/cipher.test.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  const result = cipher.encrypt("this is a test you don't suposse to read this", 1);
+  expect(result).toBe('uijt;jt;b;uftu;zpv;epo(u;tvqpttf;up;sfbe;uijt');
+});
+
+test('keeping the same case', () => {
+  const result = cipher.encrypt('PoPoTiTo', 15);
+  expect(result).toBe('EdEdIxId');
+});
+
+test('encrypting punctuation', () => {
```
Comment (Oghenebrume50) [inline test/cipher.test.js:13]: This should actually be a test that punctuations are kept not encrypted :slightly_smiling_face:

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED

Hi @mesielepush good job working on this :+1: everything checks out properly except for a little issue with the cipher encryption. 

 - You do want to preserve special characters, so, for example, is there is a . or a - in the word you want to return everything else encrypted except the . and the - still in the right position :slightly_smiling_face: 

**Please make changes and request for another review** :slightly_smiling_face: 

**RAPHAEL NORIODE** :mask:

**Comment by mesielepush** [comment]
Ready no more punctuation ciphering

**Review by bellom — APPROVED** [review]
## Hi @mesielepush  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**